pkg/api/validation/validation_test.go

```diff
@@ -3506,6 +3506,11 @@ func TestValidatePodSpec(t *testing.T) {
 	minGroupID := types.UnixGroupID(0)
 	maxGroupID := types.UnixGroupID(2147483647)
 
+	err := utilfeature.DefaultFeatureGate.Set("PodPriority=true")
```

Comment:
Looks like it is a common practice in our tests and in that file. Anyway, I added the code to obtain and restore the value.